inventory-generation/tower-jobs-schedules/README.md

```diff
@@ -0,0 +1,16 @@
+# Ansible Tower
+
+This playbook generates an inventory based off of a list of `engagement_users` found in the `engagement.json` file. It parses out the users into the expected structure found [here](https://github.com/redhat-cop/infra-ansible).
```

Comment:
Needs an update to reflect this implementation? (references the `engagement_users`, etc. which isn't applicable here)